Dockerfile

```diff
@@ -1,16 +1,22 @@
 FROM python:3.10.8
 WORKDIR /home/pandas
 
-RUN apt-get update && apt-get -y upgrade
-RUN apt-get install -y build-essential bash-completion
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    rm -rf /var/lib/apt/lists/*
 
-# hdf5 needed for pytables installation
-# libgles2-mesa needed for pytest-qt
-RUN apt-get install -y libhdf5-dev libgles2-mesa-dev
+RUN apt-get update && apt-get install -y \
```

Comment:
1. Can you combine these two installs?
2. Can you include `--no-install-recommends`?

Reply:
Yes, done. That saves another ~0.16 Gb.

Do note that the --no-install-recommends could impact  'downstream' images based on this image, if they depend on having a recommended library installed.